### PR TITLE
fixing merge conflicts

### PR DESCRIPTION
**Pull request by alextcedwards**
Please enter the commit message for your changes. Lines starting

## Description

Styled landing page according to the stakeholders mockup, created all graphics from mock up, adding them to an `img` folder in the src file on the project. 

Fixes # (issue)

## Loom Video

https://www.loom.com/share/5a3a6694b6c5490a8ba6af02f77c4b52

### Co-authored by:
@ElchinNasirov 
@JesusNavarrete215 

Please delete options that are not relevant.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by jonahjtr — CHANGES_REQUESTED** [review]
lets put some alt text for the images, to appease lint, but also for accessibility

**Review by pjaepole — APPROVED** [review]
Awesome landing page that matches the mock design, 
I think only thing it needs is margin on both left and right and it will be perfect
PR review

I believe you can achieve this by applying change to css .landing-container class
width:100%;
position: relative;
max-width: 1400px; (can be responsive unit such as % vw)
margin-left: auto;
margin-right: auto;

**Review by Ryan-E-Mark — APPROVED** [review]
Looks great, once the media queries are in place for mobile this should be complete! Great work matching the mock up.

**Review by jonahjtr — APPROVED** [review]

File: src/components/pages/Landing/RenderLandingPage.js [hunk 1]
```diff
@@ -1,56 +1,81 @@
+import ButtonGroup from 'antd/lib/button/button-group';
+import { DownOutlined } from '@ant-design/icons';
 import React from 'react';
 // import { Link } from 'react-router-dom';
```
Comment (RoseGabriely) [inline src/components/pages/Landing/RenderLandingPage.js:4]: Mind removing this just to clean up the file?

File: src/styles/ParentStyles/index.less [hunk 2]
```diff
@@ -316,135 +316,271 @@ img {
...
-    align-items: center;
-    justify-content: flex-end;
-    height: 80vh;
-    // border: red solid 2px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-end;
+  height: 280rem;
+  // border: red solid 2px;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:348]: Mind removing this commented-out code?

**Review by nforman91 — APPROVED** [review]
This looks great! One very minor catch in the RenderLandingPage component: I think "underserved" as one word is the correct syntax.

File: src/styles/ParentStyles/index.less [hunk 3]
```diff
@@ -316,135 +316,271 @@ img {
...
 .landing-main-text {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  // border: red solid 2px;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:357]: Mind removing this commented-out code?

File: src/styles/ParentStyles/index.less [hunk 4]
```diff
@@ -316,135 +316,271 @@ img {
...
+  font-weight: bold;
+  color: #3b0e2c;
+}
+
+.landing-main-container img {
+  transform: rotate(-1deg);
+  width: 65rem;
+//   position: absolute;
+//   left: 5rem;
+//   top: 20rem;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:383]: Mind removing this commented-out code?

File: src/styles/ParentStyles/index.less [hunk 5]
```diff
@@ -316,135 +316,271 @@ img {
...
-    font-size: 1rem;
-    text-align: center;
-    overflow-wrap: break-word;
-    inline-size: 600px;
-    margin: 2rem 0;
-    width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  // flex-wrap: wrap;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:393]: Mind removing this commented-out code?

File: src/styles/ParentStyles/index.less [hunk 6]
```diff
@@ -316,135 +316,271 @@ img {
...
+  font-size: 1.5rem;
+  text-align: center;
+  overflow-wrap: break-word;
+  inline-size: 37.5rem;
+//   margin: 2rem 0;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:398]: Mind removing this commented-out code?

File: src/styles/ParentStyles/index.less [hunk 7]
```diff
@@ -316,135 +316,271 @@ img {
...
-    flex-direction: row;
-    justify-content: space-between;
-    margin-top: 5rem;
-    // border: red solid 2px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: #156a70;
+  height: 200vh;
+  // border: red solid 2px;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:440]: Mind removing this commented-out code?

File: src/styles/ParentStyles/index.less [hunk 8]
```diff
@@ -316,135 +316,271 @@ img {
...
+.landing-third-container-text {
+  position: absolute;
+  padding-top: 6.25rem;
+  padding-left: 12.5rem;
+  margin-top: -3rem;
+}
+.description {
+  display: flex;
+//   justify-content: center;
+//   align-items: center;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:522]: Mind removing this commented-out code?

File: src/styles/ParentStyles/index.less [hunk 9]
```diff
@@ -316,135 +316,271 @@ img {
...
+  font-weight: bold;
+  padding-left: 1.875rem;
+  padding-top: 1.25rem;
+}
+
+.brianne {
+  display: flex;
+  flex-direction: column;
+//   justify-content: center;
+//   align-items: center;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:542]: Mind removing this commented-out code?

File: src/styles/ParentStyles/index.less [hunk 10]
```diff
@@ -316,135 +316,271 @@ img {
...
+  width: 30rem;
+  margin-top: 1.25rem;
+  margin-left: 50rem;
+  padding-left: 1.875rem;
+  color: #3b0e2c;
+  font-size: 1rem;
+  background-color: #eeedd9;
+  border: black 0.031rem solid;
+  border-radius: 2.4rem;
+  // text-align: center;
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:553]: Mind removing this commented-out code?

File: src/styles/global.less [hunk 11]
```diff
@@ -143,6 +143,7 @@ align-items: center;
     padding: 30px 0px 30px 0px;
     z-index: 999;
     position: absolute;
+
```
Comment (RoseGabriely) [inline src/styles/global.less:146]: Mind removing this extra line?

**Review by RoseGabriely — CHANGES_REQUESTED** [review]
Awesome work on this thank you! Just some clean up to do.

File: src/styles/global.less (hunk 11 above)
Comment (alextcedwards) [reply src/styles/global.less:146]: will get to all of these! looks like these were from previous cohort but I will clean them up

**Review by jonahjtr — APPROVED** [review]
great fixes, Maybe we can work on making the PR's a little smaller, though I understand sometimes touching a lot of different pages are necessary

**Review by RoseGabriely — APPROVED** [review]

**Pull request merged**